Source/Plugins/Core/com.equella.core/js/__tests__/tsrc/search/components/SearchResult.test.tsx

```diff
@@ -52,4 +53,30 @@ describe("<SearchResult/>", () => {
       queryByLabelText(container, keywordFoundInAttachmentLabel)
     ).not.toBeInTheDocument();
   });
+
+  it.each([
+    ["singular comment", mockData.basicSearchObj],
+    ["plural comments", mockData.attachSearchObj],
+  ])(
+    "should show comment count as a link for %s",
+    (testName: string, item: OEQ.Search.SearchResultItem) => {
+      const { container } = renderSearchResult(item);
```

Comment:
Better to just go straight to destructing what you want:

```suggestion
      const { getByText } = renderSearchResult(item);
```